hare/eligibility/oracle.go

```diff
@@ -575,9 +574,6 @@ func (o *Oracle) actives(ctx context.Context, targetLayer types.LayerID) (map[ty
 		activeMap[atxHeader.NodeID()] = atxHeader.GetWeight()
 	}
 	logger.With().Debug("got tortoise active set", log.Int("count", len(activeMap)))
-
-	// update cache and return
-	o.activesCache.Add(targetLayer.GetEpoch(), activeMap)
```

Comment:
@dshulyak 
this key `targetLayer.GetEpoch()` is different from the key used for hare active set `safeLayerStart.GetEpoch()`. this means the tortoise active set can pollute the data for hare active set, since `targetLayer` is always higher than `safeLayerStart`. that's why this cached value was never used in L445 because the epoch value is different.
i.e. the tortoise active set is repeatedly calculated (3 or 4 times per oracle Validate() call) until epoch 9 before the config fix in this PR.

i've considered creating a separate cache for tortoise active set. but now that the config is fixed, only genesis epoch will fall back on tortoise active set. and it seems pointless to create a cache for that.